.github/workflows/docs-preview-links.yml

```diff
@@ -6,8 +6,13 @@ on:
     paths:
       - '**.asciidoc'
 
+permissions:
+  contents: read
```

Comment:
Is this necessary? Also it's not clear to me how this stacks with job-specific permissions.

Reply:
The idea is to have restrictive top level permissions in case later someone adds another job without permissions, but it is optional. Job specific permissions replace the global ones, not merge them.

Reply:
I'm ok with removing this then as I find it kind of confusing given there's only one job. I think if we were to keep this the preference would be `permissions: {}` which I believe basically means "nothing". That seems like a safer default.

Reply:
Done.